src/main/java/com/actions/emislabbackend/model/OperationTeeth.java

```diff
@@ -0,0 +1,106 @@
+package com.actions.emislabbackend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.text.DecimalFormat;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class OperationTeeth {
+
+    private int id;
+    private String medic;
+    private String patient;
+    private WorkType worktype;
+    private int teethNr;
+
+
+    public OperationTeeth(int id, String medic, String patient, WorkType worktype, int teethNr) {
+        this.id = id;
+        this.medic = medic;
+        this.patient = patient;
+        this.worktype = worktype;
+        this.teethNr = teethNr;
+        calculateProcent(this.worktype, this.teethNr);
+    }
+
+    private double timeea;
```

Comment:
instance variables should never be under a method. The class should start with them